File: Equal-Sides-Of-An-Array/0.java
public class Kata {
    public static int findEvenIndex(int[] arr) {
        int lsum = 0, rsum = 0;
        for (int num : arr) rsum += num;
        for (int i = 0; i < arr.length; ++i) {
            rsum -= arr[i];
            if (lsum == rsum) return i;
            lsum += arr[i];
        }
        return -1;
    }
}

File: 5679aa472b8f57fb8c000047/0.c
int find_even_index(const int* values, int length) {
    int lsum = 0, rsum = 0;
    for (int i = 0; i < length; ++i) rsum += values[i];
    for (int i = 0; i < length; ++i) {
        rsum -= values[i];
        if (lsum == rsum) return i;
        lsum += values[i];
    }
    return -1;
}
